.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -123,7 +123,7 @@ jobs:
           extra_system_packages: "inkscape font-liberation font-hack"
           latexmk_use_lualatex: ${{ matrix.luaengine }}
           latexmk_shell_escape: true
-          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && pdflatex --shell-escape main_${{ matrix.language }}.tex
+          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && lualatex --shell-escape main_${{ matrix.language }}.tex
```

Comment:
Are you sure about this? Only one file builds with lualatex

Reply:
oops